scripts/backup

```diff
@@ -6,7 +6,6 @@
 # IMPORT GENERIC HELPERS
 #=================================================
 
-#Keep this path for calling _common.sh inside the execution's context of backup and restore scripts
```

Comment:
I usually keep that comment as a warning for anyone wanted to work on the package